net/DevExtreme.AspNet.Data/Aggregation/CountAggregator.cs

```diff
@@ -7,9 +7,15 @@ namespace DevExtreme.AspNet.Data.Aggregation {
 
     class CountAggregator : Aggregator {
         int _count;
+        bool _skipNulls;
+
+        public CountAggregator(bool skipNulls) {
+            _skipNulls = skipNulls;
+        }
 
         public override void Step(object value) {
-            _count++;
+            if(!_skipNulls || value != null)
```

Comment:
Since the `count` aggregate calculation was optimized (we just get the count of items and don't call the step function), it always ignores the `_skipNull` flag and returns the real count of items.